tests/wallet/rpc/test_wallet_rpc.py

```diff
@@ -84,7 +84,7 @@ async def farm_transaction_block(full_node_api: FullNodeSimulator, wallet_node:
 
 
 def check_mempool_spend_count(full_node_api: FullNodeSimulator, num_of_spends):
-    return len(full_node_api.full_node.mempool_manager.mempool.sorted_spends) == num_of_spends
```

Comment:
this is the other example of changing from counting *levels* to number of items

Reply:
Nice. `num_of_spends` and `sorted_spends` are both misleading here as well.